docs/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next",
     "start": "next start",
     "build": "next build ",
-    "lint": "next lint --max-warnings 356",
+    "lint": "eslint . --max-warnings 293 --fix",
```

Comment:
This is purposeful for now. It shows the error count at the end of the script's run so I know what the new number is.